Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -29,6 +29,9 @@ import BraintreeDataCollector
     /// Exposed for testing, the ASWebAuthenticationSession instance used for the PayPal flow
     var webAuthenticationSession: BTWebAuthenticationSession
 
+    /// True if `tokenize()` was called with a VaultRequest object type
+    var isVaultRequest: Bool = false
```

Comment:
I know this wasn't mentioned in the ticket, but can we also set this property for the VenmoRequests in VenmoClient? Also all good if you want to make a new ticket for that

Reply:
I can make the change in this PR! Do you mean updating the parameter name from `request` to `venmoRequest`? or create a new internal property `var isVenmoRequest: Bool` in `BTVenmoRequest`?

Reply:
I think what Sammy is saying is to add a property `isVaultRequest` to `BTVenmoClient` and pass it as `true/false` in that flow as well. I don't think it needs to be renamed or anything, just passed in the `BTVenmoClient` for vaulting/non-vaulting flows.